fluentd/run.sh

```diff
@@ -31,6 +31,10 @@ if [ "${USE_JOURNAL:-}" != false ] ; then
     fi
 fi
 
+IPADDR4=`/usr/sbin/ip -4 addr show dev eth0 | grep inet | sed -e "s/[ \t]*inet \([0-9.]*\).*/\1/"`
```

Comment:
I'm actually getting SDN's pod IP for IPADDR4(same for IPADDR6). My expectation would be that I get node's IP, however I don't see node's interface from inside the pod.
That means `ipaddr4`/`ipaddr6` fields should be moved under `pipeline_metadata.collector`